test/integration/open-scd.test.ts

```diff
@@ -195,7 +195,7 @@ describe('open-scd', () => {
     await element.validated;
     expect(element).property('history').to.have.length(4);
     expect(element.doc?.querySelector('Bay[name="COUPLING_BAY"]')).to.exist;
-  });
+  }).timeout(10000);
```

Comment:
Why is there a timeout added?

Reply:
Done by Christian in OpenSCD, because the test are taking longer then the default.